eclipse-scout-core/src/action/Action.ts

```diff
@@ -286,14 +290,14 @@ export class Action extends Widget implements ActionModel {
     } else {
       tooltips.uninstall(this.$container);
     }
-    aria.description(this.$container, this.tooltipText);
+    aria.description(this.$container, this._tooltipText());
```

Comment:
Maybe screen readers now read the same text twice because the aria-label and description contain the text. I am not sure about that, needs to be tested. Maybe we should only add the description if it differs from the text.

Reply:
Correct. Aria attributes should not be affected by the shrinked state.